gateway/server.go

```diff
@@ -169,9 +169,14 @@ func setupGlobals(ctx context.Context) {
 		mainLog.Fatal("Analytics requires Redis Storage backend, please enable Redis in the tyk.conf file.")
 	}
 
-	// Initialise our Host Checker
-	healthCheckStore := storage.RedisCluster{KeyPrefix: "host-checker:"}
-	InitHostCheckManager(&healthCheckStore)
+	if config.Global().UptimeTests.DisableManagementPoller && config.Global().ManagementNode {
```

Comment:
This seems a bit iffy.

The log here should state the opposite I think because we are saying that disabling the management poller is true.

Also could this cause panics down the line if the store is not initialised but uptime checks are still enabled somewhere?

Reply:
@joshblakeley updated the warning condition for managementNode = true and DisableManagementPoller=false.

I also tested the config with uptime_tests.disable_management_poller = true and uptime_tests.disable = false and there are no problems with panics.